ibm/service/kubernetes/resource_ibm_container_vpc_worker.go

```diff
@@ -65,6 +71,46 @@ func ResourceIBMContainerVpcWorker() *schema.Resource {
 				Description: "Cluster name",
 			},
 
+			"sds": {
```

Comment:
"sds" & "kube_config_path" parameter are dependent to each other. So update the schema with "RequiredWith" also.